rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -208,6 +208,7 @@ dfs knn vector profiling:
   - match: { profile.shards.0.dfs.knn.0.query.0.type: "DocAndScoreQuery" }
   - match: { profile.shards.0.dfs.knn.0.query.0.description: "DocAndScore[100]" }
   - gt: { profile.shards.0.dfs.knn.0.query.0.time_in_nanos: 0 }
+  - match: { profile.shards.0.dfs.knn.0.vector_operations_count: 2 }
```

Comment:
I think we might need to check version for this test, because `vector_operations_count` won't exist in previous versions

Reply:
Updated